### ISSUE TITLE
Multi-Display Support (window sizes)

### ISSUE THREAD

**Issue by gdw2** [paulswartz/gnome-shell-grid#2]
This issue is actually different than the other with the same title...

I've got three displays.  When I have a window in the middle display, and send it to the left, it goes to the left screen and fills it up.  I would expect it to go to the left side of the current (middle) screen.  hen I send it to the right, it goes to my right screen.  

It's obvious that it must think I have one giant screen instead of three smaller screens.

**Comment by paulswartz** [comment]
Yeah, it's just grabbing the size of the whole desktop, and splitting against that.  Unfortunately, I no longer have a dual screen setup, so there's not much I can do about this at the moment.  If you file a pull request with a fix, though, I'd love to include it!
